emily_sidecar/test.py

```diff
@@ -0,0 +1,75 @@
+import os
+import unittest
+from unittest.mock import patch, MagicMock
+import os
+
+import json
+import requests
+
+from app import app
+
+
+class NewBlockTestCase(unittest.TestCase):
+    def setUp(self):
+        self.app = app.test_client()
+        self.app.testing = True
+
+    def read_fixture(self, filename):
+        with open(filename, 'r') as file:
+            return json.load(file)
+
+    @patch('requests.post')
+    def test_new_block_valid_json(self, mock_post):
+        # Mock the response from requests.post
+        mock_response = MagicMock()
+        mock_response.status_code = 200
+        mock_response.json.return_value = {"message": "Success"}
+        mock_post.return_value = mock_response
+
+        base_path = os.path.join(os.path.dirname(__file__), "..")
+        fixtures_path = os.path.join(base_path, "signer", "tests", "fixtures")
```

Comment:
nit: move `fixtures_path` to global const? since we use it also in `test_new_block_post_request_failure`

Reply:
I refactored the fixtures part to be global in https://github.com/stacks-network/sbtc/pull/1050/commits/eac2cdfad8bc01ead03d70ecc0b9e0c0714036e2